File: api/code-samples/python/klay/others/getRawTransactionByHash.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

hash = "0x29b6cd965c7d9a53a6f068da259dce1d3810ba79fff8eebac5d4da14754e67e6"

sdk = OpenSDK(host)
klay_response = sdk.klay.get_raw_transaction_by_hash(hash)

print(json.loads(klay_response.response.data))

File: web3rpc/rpc-specs/code-samples/javascript/klay/transaction/getRawTransactionByHash.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

  const transactionHash =
    "0x21b2919b89278ca786226f10edbaadced7381dbd73df546a4823547aaebffa58";

  provider.klay
    .getRawTransactionByHash(transactionHash, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
